.github/workflows/ci.yml

```diff
@@ -78,17 +78,15 @@ jobs:
 
       - name: install class
         run: |
-          export LDFLAGS=-L/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib
           if [[ ${MATRIX_OS} == "macos-11" ]]; then
+            export LDFLAGS=-L/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib
+            LDFLAGS+=" -L/Users/runner/miniconda3/envs/test/lib"
```

Comment:
Don't you need to do
```
export LDFLAGS+=" -L/Users/runner/miniconda3/envs/test/lib"
```

Reply:
I also tried with zsh and I got error when doing `export LDFLAGS+=" -L/Users/runner/miniconda3/envs/test/lib"`

Reply:
interestingly, it is okay with bash.

Reply:
I am using something that works for both shells.

Reply:
Interesting! OK, as long as it works I'm happy.